message-service/src/main/java/com/codecool/messageservice/service/MailService.java

```diff
@@ -0,0 +1,36 @@
+package com.codecool.messageservice.service;
+
+import com.codecool.messageservice.consumer.dto.OrderDTO;
+import com.codecool.messageservice.consumer.dto.OrderStatusChange;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.mail.SimpleMailMessage;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class MailService {
+    private final JavaMailSender javaMailSender;
+
+    @Value("${spring.mail.username}")
+    private String from;
+
+    public void sendOrderStatusMail(OrderStatusChange orderStatusChange) {
+        OrderDTO order = orderStatusChange.order();
+
+        final SimpleMailMessage simpleMailMessage = getStatusMail(orderStatusChange, order);
+
+        javaMailSender.send(simpleMailMessage);
+    }
+
+    private SimpleMailMessage getStatusMail(OrderStatusChange orderStatusChange, OrderDTO order) {
```

Comment:
I'd like to name it "buildStatusMail or something like that. This implies that it gets a mail from DB or other source.